### ISSUE TITLE
Coherent use of canvases (if that's the plural for canvas)

### ISSUE THREAD

**Issue by oas777** [opencast/opencast-editor#1106]
Complementary and preparatory with my suggestion for coherent "[elements](https://github.com/opencast/opencast-editor/pull/1079#issuecomment-1629168742)" I suggest to coherently use the white canvas for video(s): 

https://test.editor.opencast.org/1079/2023-07-17_14-50-34/ has a white canvas for each video in "Spuren" and "Vorschaubild", but not in "Schneiden".

**Comment by Arnei** [comment]
Actually, by now I think I overdid it with the boxing of elements in the "Thumbnail/Vorschaubild" view. So instead of coherently using white canvases everywhere, how about coherently not using them?

![Bildschirmfoto vom 2023-07-18 09-10-38](https://github.com/opencast/opencast-editor/assets/14070005/a297cfa2-fd08-4ef7-8742-eee290af229f)

And I'm differentiating between Video Player elements (in e.g. "Schneiden/Cutting") and preview image elements (in e.g. "Spuren/Tracks"). I still think it is fine and possibly beneficial to treat them differently.

**Comment by oas777** [comment]
In line with https://github.com/opencast/opencast-editor/issues/1107#issuecomment-1641059970, I would argue in favor of separating the two videos and give them a white canvas each to match the previews above. Once they are aligned, you might try a "non-canvas" design so we can decide how to move forward.

**Comment by lkiesow** [comment]
I hope you don't actually mean what you wrote down.
The dark mode in your example, @oas777, has no white canvas as far as I can tell.
Adding one would probably look pretty bad.

![Screenshot from 2023-08-10 12-37-58](https://github.com/opencast/opencast-editor/assets/1008395/ca1e005f-9742-4257-b5a7-53d5c9745ce9)

So, I think/hope you mean a neutral canvas based on the color mode?

**Comment by oas777** [comment]
Yes, I was obviously commenting in the context of the custom mode, where white makes sense to me. What's your suggestion for the dark mode?

**Comment by Arnei** [comment]
Copying over some screenshots Olaf took of the current state:

![grafik](https://github.com/opencast/opencast-editor/assets/1752342/79d2c508-8a01-484a-a592-11d5f14ff67a)
vs.
![grafik](https://github.com/opencast/opencast-editor/assets/1752342/6804f5d6-09c2-46de-9527-f525b98cb9b9)
vs.
![grafik](https://github.com/opencast/opencast-editor/assets/1752342/1fc21bc0-8a78-40c9-ac5b-d5803cc9652e)

**Comment by Arnei** [comment]
While I can see how having a track title (i.e. the track main flavor) in every canvas would be overall more consistent, I'm personally not in favour of it. I don't think we should use space to display information that is not relevant to the task at hand. For example, when cutting it is not important to me what the tracks main flavors are, unlike when selecting a track where the preview image may not be sufficient to differentiate between tracks.

But that's just my opinion, I would like to hear others opinions on this.

**Comment by oas777** [comment]
@lkiesow?
